frontend/www/js/omegaup/components/arena/Virtual.vue

```diff
@@ -75,7 +75,7 @@ export default class ArenaVirtual extends Vue {
   virtualContestStartTime = new Date();
 
   onSubmit(): void {
-    this.$emit('submit', this);
+    this.$emit('submit', this.virtualContestStartTime);
```

Comment:
```suggestion
    this.$emit('submit', { virtualContestStartTime: this.virtualContestStartTime });
```

por consistencia con los demás lugares.

Reply:
Listo.